documentation/modules/configuring/proc-config-mirrormaker2-replication.adoc

```diff
@@ -34,14 +34,26 @@ spec:
   version: {DefaultKafkaVersion}
   connectCluster: "my-cluster-target"
   clusters:
-  - alias: "my-cluster-source"
-    bootstrapServers: my-cluster-source-kafka-bootstrap:9092
-  - alias: "my-cluster-target"
-    bootstrapServers: my-cluster-target-kafka-bootstrap:9092
+  - alias: "sourceCluster"
+    bootstrapServers: my-source-cluster-bootstrap:9094
+    config:
+      config.storage.replication.factor: 1
+      offset.storage.replication.factor: 1
+      status.storage.replication.factor: 1
+  - alias: "targetCluster"
+    bootstrapServers: my-target-cluster-bootstrap:9092
+    config:
+      producer.compression.type: zstd
```

Comment:
As I said before. The easiest way to add this would be to add a small paragraphs saying something like _If you want to use compression to ...._ And add a separate snippet with this to it.